utils/create_admin_user.py

```diff
@@ -2,39 +2,69 @@
 import re
 
 from forms.custom_validators import email_regex
+from ldap_utils.ldap_utils import ldap_client, refine_data
 from models.users import User, Role, RoleEnum
 from init_db import db
 
 
-def create_super_user():
-    email_data = input("What is your email: ")
-    print("Introduce yourself:\n")
-    first_name = input("First name: ")
-    surname = input("Surname: ")
-    password = input("Password: ")
-    password_2 = input("Password: ")
-    if User.query.filter_by(email=email_data).first():
-        return print("This e-mail already exists in the database!")
-    if not re.compile(email_regex()).match(email_data):
-        return print("Only Tieto emails are accepted!")
-    if password != password_2:
-        return print("Passwords are not the same!")
-    new_user = User(
-        email=email_data,
-        first_name=first_name,
-        surname=surname,
-        password_hash=generate_password_hash(
-            password),
-        active=True
+email_list = [
+    'katarzyna.kaminska@tieto.com',
+    'hubert.nafalski@tieto.com',
+    'pawel.biel@tieto.com',
+    'lukasz.jaszewski@tieto.com'
+]
```

Comment:
I think it's not good idea, because in some time these employees may not work in this company.
Secondly, GPRD (RODO). 
Thirdly hardcoding emails in code it's bad practice